tests/pairlist/test_pairlist.py

```diff
@@ -202,6 +203,9 @@ def test_VolumePairList_refresh_empty(mocker, markets_empty, whitelist_conf):
 
 
 @pytest.mark.parametrize("pairlists,base_currency,whitelist_result", [
+    # Empty pairlist
+    ([],
```

Comment:
You'll never get this far if you initialize the bot correctly (not from within a test, skipping most of the initialization part) - as we do add StaticPairList if the list is empty.

So for me, this is (at least at the moment) a pointless test - as no user can realistically reach this point.

Reply:
right, I'll remove this test case